### ISSUE TITLE
Wrong Val acc?

### ISSUE THREAD

**Issue by hawklucky** [hx173149/C3D-tensorflow#2]
Thanks for your excellent work! When I trained the model at around 10000 iters, the batch acc of training set is ~96%, but the batch acc of val set is ~ 29%, what's wrong? the training list and val list is the list i used for training in CNN which works well.

**Comment by hx173149** [comment]
Thanks for telling me this problem, I actually didn't make a validation in UCF101 dataset.
I just use this code to a binary classification problem, it works......
I will check this problem after 10.09, I am on my vocation now......
cheers~

**Comment by hx173149** [comment]
@hawklucky Hi I have also found this bug in my experienment, when I get this train accuracy(93.75%):
![image](https://cloud.githubusercontent.com/assets/5306116/19243490/719451f0-8f4a-11e6-8e34-439d059e1e4d.png)
I can only get a low val accuracy(32.8%):
![image](https://cloud.githubusercontent.com/assets/5306116/19243602/c6d34298-8f4a-11e6-932f-f7c00aac7864.png)

I think there must be some over fitting problem, maybe some bad parameter like L2 regular, ExponentialMovingAverage......
I will check where is the problem next days, if you find some bugs you can tell me too, thanks~

**Comment by hx173149** [comment]
@hawklucky recently, I have solved this problem. And if you want to get a high acc as same as the caffe version you must fine-tuning from the author's sports1m model. So I have transferred the caffe model to TF model, and fine-tuning this model from UCF101 dataset, and get a 72% acc.
